examples/module/src/module_arm/module_arm.py

```diff
@@ -0,0 +1,96 @@
+import asyncio
+import os
+from typing import Any, ClassVar, Dict, Mapping, Optional, Self, Tuple
+from viam.components.arm import Arm, JointPositions, KinematicsFileFormat, Pose
+from viam.operations import run_with_operation
+from viam.proto.app.robot import ComponentConfig
+from viam.proto.common import ResourceName
+from viam.resource.base import ResourceBase
+from viam.resource.types import Model, ModelFamily
+
+
+class ModuleArm(Arm):
```

Comment:
would name it something else - MyArm or CustomArm, since it being a modular arm is not very descriptive